[2023-06-16T06:07:00.960+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ejecutar_python.check_participants manual__2023-06-15T14:47:07.980597+00:00 [queued]>
[2023-06-16T06:07:00.967+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ejecutar_python.check_participants manual__2023-06-15T14:47:07.980597+00:00 [queued]>
[2023-06-16T06:07:00.967+0200] {taskinstance.py:1308} INFO - Starting attempt 28 of 28
[2023-06-16T06:07:01.002+0200] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): check_participants> on 2023-06-15 14:47:07.980597+00:00
[2023-06-16T06:07:01.007+0200] {standard_task_runner.py:57} INFO - Started process 248942 to run task
[2023-06-16T06:07:01.010+0200] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'ejecutar_python', 'check_participants', 'manual__2023-06-15T14:47:07.980597+00:00', '--job-id', '72', '--raw', '--subdir', 'DAGS_FOLDER/thesis_container.py', '--cfg-path', '/tmp/tmp0fh15ixr']
[2023-06-16T06:07:01.012+0200] {standard_task_runner.py:85} INFO - Job 72: Subtask check_participants
[2023-06-16T06:07:01.093+0200] {task_command.py:410} INFO - Running <TaskInstance: ejecutar_python.check_participants manual__2023-06-15T14:47:07.980597+00:00 [running]> on host DESKTOP-3KJU1M7.
[2023-06-16T06:07:01.195+0200] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='ejecutar_python' AIRFLOW_CTX_TASK_ID='check_participants' AIRFLOW_CTX_EXECUTION_DATE='2023-06-15T14:47:07.980597+00:00' AIRFLOW_CTX_TRY_NUMBER='28' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-15T14:47:07.980597+00:00'
[2023-06-16T06:07:01.197+0200] {logging_mixin.py:149} INFO - ['hola_525']
[2023-06-16T06:07:01.197+0200] {logging_mixin.py:149} INFO - ['']
[2023-06-16T06:07:01.197+0200] {logging_mixin.py:149} INFO - ['hola_525']
[2023-06-16T06:07:01.197+0200] {logging_mixin.py:149} INFO - New participants found. Creating containers...
[2023-06-16T06:07:01.197+0200] {logging_mixin.py:149} INFO - 0
[2023-06-16T06:07:01.197+0200] {logging_mixin.py:149} INFO - <class 'int'>
[2023-06-16T06:07:01.197+0200] {logging_mixin.py:149} INFO - ['hola_525']
[2023-06-16T06:07:01.198+0200] {logging_mixin.py:149} INFO - prueba de lista ['hola_525']
[2023-06-16T06:07:01.198+0200] {logging_mixin.py:149} INFO - ['/home/reynel1995/Thesis/host_2/1_6-6.mrxs']
[2023-06-16T06:07:01.198+0200] {logging_mixin.py:149} INFO - container solito hola_525
[2023-06-16T06:07:01.198+0200] {logging_mixin.py:149} INFO - /home/reynel1995/Thesis/host_2/1_6-6.mrxs
[2023-06-16T06:07:01.198+0200] {logging_mixin.py:149} INFO - /home/reynel1995/Thesis/host_2/1_6-6.mrxs
[2023-06-16T06:07:01.198+0200] {logging_mixin.py:149} INFO - /home/reynel1995/Thesis/host_2
[2023-06-16T06:07:01.198+0200] {logging_mixin.py:149} INFO - nuevo /home/reynel1995/Thesis/hola_525
[2023-06-16T06:07:01.198+0200] {logging_mixin.py:149} INFO - ['/home/reynel1995/Thesis/hola_525', '', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525']
[2023-06-16T06:07:05.241+0200] {logging_mixin.py:149} INFO - Container hola_525 image built successfully.
[2023-06-16T06:07:05.408+0200] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/reynel1995/thesis/lib/python3.10/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/reynel1995/thesis/lib/python3.10/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/reynel1995/airflow/dags/thesis_container.py", line 327, in check_participants
    build_and_run_containers(index)
  File "/home/reynel1995/airflow/dags/thesis_container.py", line 144, in build_and_run_containers
    subprocess.run(run_command, shell=True, check=True)
  File "/usr/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command 'docker run -d --name hola_525 hola_525' returned non-zero exit status 125.
[2023-06-16T06:07:05.416+0200] {taskinstance.py:1345} INFO - Marking task as FAILED. dag_id=ejecutar_python, task_id=check_participants, execution_date=20230615T144707, start_date=20230616T040700, end_date=20230616T040705
[2023-06-16T06:07:05.506+0200] {standard_task_runner.py:104} ERROR - Failed to execute job 72 for task check_participants (Command 'docker run -d --name hola_525 hola_525' returned non-zero exit status 125.; 248942)
[2023-06-16T06:07:05.523+0200] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-16T06:07:05.557+0200] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
